packages/@aws-cdk/aws-s3-deployment/test/bucket-deployment.test.ts

```diff
@@ -325,6 +325,7 @@ test('system metadata is correctly transformed', () => {
     websiteRedirectLocation: 'example',
     cacheControl: [s3deploy.CacheControl.setPublic(), s3deploy.CacheControl.maxAge(cdk.Duration.hours(1))],
     expires: expiration,
+    accessControl: s3.BucketAccessControl.BUCKET_OWNER_FULL_CONTROL,
```

Comment:
can you add a separate test case for each type of `BucketAccessControl`, so we can ensure that all types are correctly transformed by `toKebabCase()`?

Reply:
I have added the test.